TUTORIAL_INSTALL.md

```diff
@@ -0,0 +1,106 @@
+# Tutorial: Installing Typedefs
+
+This tutorial aims to explain how to install and use _Typedefs_. _Typedefs_ is a language to define _types_ and 
+generate type definitions as well as serializers and deserializers for a target language. We are going to see later how 
+to achieve that, right now we are going to focus on how to install the tools.
+
+## 0. Install Idris
+
+Since _Typedefs_ is written using Idris we are going to install the [Idris](IDRIS_HOME) compiler. You can find 
+instruction on how to install Idris for your platform here: [https://www.idris-lang.org/download/][IDRIS_DOWNLOAD].
+
+
+#### Shortcut: Mac OS
+
+If you're using Mac OS, you can use [`brew`][BREW] to install Idris by typing this on your terminal
```

Comment:
Experience report: Brew hasn't worked for me in years; macports has been broken for a long time too, probably because of OS upgrades on my machine or something. Nix did work for me though; I think I used `nix-env -iA idris`, but neither that nor the suggestions given at https://nixos.org/nixos/packages.html#idris seem to work for me now. (Maybe i have some nixos/nixpkgs channel I have to select or something, not sure, but I could probably find out.

Reply:
Should we just remove the mention to brew then?